mobile/pubspec.yaml

```diff
@@ -17,7 +17,7 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 version: 2.0.20+20036
 
 environment:
-  sdk: '>=2.18.0 <3.0.0'
+  sdk: '>=2.19.0 <3.0.0'
```

Comment:
Hi @NoahNsimbe , I've made this change because i was getting this error. `This API is available since SDK 2.19.0, but constraints '>=2.18.0 <3.0.0' don't guarantee it.`` but the app still runs fine